src/plugins/sql-support/plugin.js

```diff
@@ -389,9 +389,14 @@ QueryBuilder.extend({
                 var opVal = sqlrl.call(this, value, data.operation);
                 if (operator == 'NOT LIKE') opVal.op = 'not_' + opVal.op;
 
+                var left_value = data.left.value.value;
```

Comment:
I can't test right now but I highly doubt you are right, it worked perfectly before (for simple fields names) with `data.left.value`